### ISSUE TITLE
Nanoboy advance core

### ISSUE THREAD

**Issue by GFLJS2100** [libretro/RetroArch#16597]
Add the nanoboy advance core

**Comment by zoltanvb** [comment]
Just for reference, upstream repo is I guess this one:
https://github.com/nba-emu/NanoBoyAdvance
There is no libretro core in the original repo, there is an unofficial port, but seems abandoned since a few years:
https://github.com/kremi151/NanoboyAdvance-libretro

**Comment by hizzlekizzle** [comment]
Yes, the upstream project changed structure quite a bit since the unofficial port was made, so it's not particularly useful, I don't think. Probably some of the function names are the same, but that's likely the extent of it.

This would probably be a good target to libretro-ize, though, since upstream is no longer under development.

**Comment by RetroGamingX** [comment]
Without a doubt the best GBA emulator, especially in audio.
